client/src/templates/Challenges/components/Side-Panel.js

```diff
@@ -37,21 +37,23 @@ export class SidePanel extends Component {
   componentDidMount() {
     const MathJax = global.MathJax;
     const mathJaxMountPoint = document.querySelector('#mathjax');
-    const rosettaCodeChallenge = this.props.section === 'rosetta-code';
+    const rosettaCodeChallenge =
+      this.props.section === 'rosetta-code' || 'project-euler';
```

Comment:
```suggestion
      this.props.section === 'rosetta-code' ||
      this.props.section === 'project-euler';
```
As written it would either be `true` or `'project-euler'` which is also truthy.

Reply:
@ojeytonwilliams, thank you for catching this. This would cause a lot of problems! I'll make the change now.